cljfmt/src/cljfmt/core.cljc

```diff
@@ -71,9 +71,90 @@
        (not (namespaced-map? (z/up* zloc)))
        (element? (z/right* zloc))))
 
+(def ^:private default-align-bindings-args
+  (read-resource "cljfmt/align_bindings/clojure.clj"))
+
+(defn- ks->max-length [ks]
+  (if (empty? ks)
+    0
+    (->> ks
+         (apply max-key (comp count str))
+         str
+         count)))
+
+
+(defn- aligner [zloc max-length align?]
+  (cond
+    (zero? max-length)  (z/up zloc)
+    (z/rightmost? zloc) (z/up zloc)
+    align?              (let [clean-zloc (-> zloc
+                                             z/right*
+                                             (z/replace (n/whitespace-node " "))
+                                             z/left)
+                              to-add     (->> clean-zloc
+                                              z/sexpr
+                                              str
+                                              count
+                                              (- max-length))
+                              new-zloc   (z/insert-space-right clean-zloc to-add)]
+                          (aligner (z/right new-zloc) max-length false))
+    :else               (aligner (z/right zloc) max-length true)))
```

Comment:
Sure, I used it to alternate between values so it works for and series of values aligning only the odds (you can see the alternation using the boolean flag).

I think your implementation in the previous PR is nicer if you can make it work, but I didn't know anything about clj-rewrite when I started working on this and I couldn't understand what you're doing there. I can go back to it later.

I think a lot of this code can be nicer. It also doesn't handle multi-binding lines.

I am trying to not take too much into this and try and leave what can be left for future improvements (including the core functions).

I ran the latest commit on cljfmt and it looks nice IMO. I rarely use a single-line for multiple bindings, I see you (and possibly others) use it more often. I opted to skip single-line maps/bindings, but it will still uglify multiline and multi-value maps and bindings.